src/components/app-blog-content/app-blog-content.scss

```diff
@@ -104,22 +104,102 @@
   }
 }
 
+.contact-icon {
+  margin: 10px;
+  img {
+    width: 20px;
+  }
+}
+
 .contact-icons-top {
   display: flex;
   justify-content: center;
-  margin-bottom: 10px;
+
+  .contact-icon {
+    margin: 5px;
+    img {
+      border-radius: 0;
+      width: 15px;
+      height: 15px;
+    }
+  }
+
+  @include media-breakpoint-down(sm) {
+    display: initial;
+    margin-bottom: 0;
+    justify-content: initial;
+  }
 }
 
 .contact-icons-side {
   display: flex;
   flex-direction: column;
   float: right;
   padding: 15px;
+  @include media-breakpoint-down(md) {
+    display: none;
+  }
 }
 
-.contact-icon {
-  margin: 10px;
-  img {
-    width: 20px;
+.author-img-container {
+  display: flex;
+  align-items: center;
+}
+
+#blog-next-posts-side {
+  display: block;
+
+  @include media-breakpoint-down(md) {
+    display: none;
+  }
+}
+
+#blog-next-posts-bottom {
+  display: none;
+
+  @include media-breakpoint-down(md) {
+    display: block;
+  }
+}
+
+.blog-next-posts {
+  h1 {
+    color: #292a2d;
```

Comment:
Do we have a global variable for this color? If not, we should probably add it for ease of use. I've seen it a few times within this file.